coursemapper-kg/recommendation/app/log.py

```
@@ -0,0 +1,66 @@
+        redis_handler = RedisHandler(worker_id, redis)
+        redis_handler.setFormatter(self.formatter)
+        return redis_handler
+
+
+class RedisHandler(Handler):
+    def __init__(self, worker_id, redis):
+        Handler.__init__(self)
+        self.worker_id = worker_id
+        self.redis = redis
+
+    def emit(self, record):
```

Ruff (0.12.2)

Missing return type annotation for public function `emit`

Add return type annotation: `None`

(ANN201)